modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesKernelBoot.java

```diff
@@ -289,15 +290,29 @@ public static void startAndesStores() throws Exception {
 
 
     /**
-     * bring the node to the state of the cluster
+     * Bring the node to the state of the cluster. If this is the coordinator, disconnect all active durable
+     * subscriptions.
      *
-     * @throws Exception
+     * @throws AndesException
      */
     public static void syncNodeWithClusterState() throws AndesException {
+
+        // Mark all the existing durable subscriptions as inactive when starting a standalone node or the coordinator
+        // node since there can be no active durable subscribers when starting the first node of the cluster.
+        // Having existing active durable subscribers causes them to not be able to reconnect to the broker
+        // The deactivation should not be performed by just any node but the first node to start
+        if (AndesContext.getInstance().isClusteringEnabled()) {
+            if (AndesContext.getInstance().getClusterAgent().isCoordinator()) {
+                ClusterResourceHolder.getInstance().getSubscriptionManager().deactivateAllActiveSubscriptions();
+            }
+        } else {
+            ClusterResourceHolder.getInstance().getSubscriptionManager().deactivateAllActiveSubscriptions();
+        }
```

Comment:
Since you need to do the same thing (ClusterResourceHolder.getInstance().getSubscriptionManager().deactivateAllActiveSubscriptions()) on both the conditions, you need to make it one condition (which improves the clarity). Use if-else only if you need to do different things on different conditions.